### PR TITLE
Test volume clerk client

### PR DESCRIPTION
**Pull request by dbox-fnal**

### CODE REVIEW

File: src/tests/test_volume_clerk_client.py
Comment (renbauer-fnal) [inline src/tests/test_volume_clerk_client.py:50]: Can we add a test case for KB here and one that sends a negative?

File: src/tests/test_volume_clerk_client.py
Comment (renbauer-fnal) [inline src/tests/test_volume_clerk_client.py:112]: Do we know what this function (my_atol) actually does? It appears to parse units to standardize values but what does "atol" mean? Why isn't it called my_size_in_bytes?

File: src/tests/test_volume_clerk_client.py
Comment (renbauer-fnal) [inline src/tests/test_volume_clerk_client.py:393]: Man, all these are almost identical. Can we simplify these functions to be something like:
```
def assert_generates_work_ticket(work_str, test_func, extra_items={}):
  expected_work_ticket = {'work': work_str,
                          'external_label': 'external_label'} + extra_items
  self.sent_msg.reset_mock()
  test_func()
  generated_work_ticket = self.sent_msg.mock_calls[0][1][0]
  self.assertEqual(generated_work_ticket, expected_work_ticket)

def test_set_system_migrated(self):
  assert_generates_work_ticket('set_system_migrates', 
     lambda: self.vcc.set_system_migrated('external_label'))
```

Maybe there is a cleaner way than passing lambdas but my kwargs-fu is not up to it atm

File: src/tests/test_volume_clerk_client.py
Comment (renbauer-fnal) [inline src/tests/test_volume_clerk_client.py]: Nice var name, but is this used?

File: src/tests/test_volume_clerk_client.py
Comment (renbauer-fnal) [inline src/tests/test_volume_clerk_client.py]: Can we move these resets to the bottom of the test cases above, like you are doing later in these cases?

**Review by renbauer-fnal — CHANGES_REQUESTED** [review]

File: src/tests/test_volume_clerk_client.py
Comment (dbox-fnal) [reply src/tests/test_volume_clerk_client.py]: Done.

File: src/tests/test_volume_clerk_client.py
Comment (dbox-fnal) [reply src/tests/test_volume_clerk_client.py]: Must be a remnant from an earlier test approach, removing does not affect the test

File: src/tests/test_volume_clerk_client.py
Comment (dbox-fnal) [reply src/tests/test_volume_clerk_client.py:393]: There are most likely ways to simplify this,  I will experiment on other repetitive client tests.  This one is done and works so I would prefer to leave it as is.  The test of do_work() lower down is also repetitive and ugly and I would like even more to come up with something parameterized for it.

File: src/tests/test_volume_clerk_client.py
Comment (dbox-fnal) [reply src/tests/test_volume_clerk_client.py:112]: My guess is named after atol(3) in the standard C library.

**Comment by dbox-fnal** [comment]
reviewer suggestions implemented, waiting for approval

**Review by renbauer-fnal — APPROVED** [review]

**Pull request merged**
